### DISCUSSION TITLE
Request: Clickable alphabetical index on web site?

### DISCUSSION THREAD

**Discussion by joe-dougherty** [navidrome/navidrome#3155] [Ideas]
I was advised to add this here after posting it on Reddit. Hope this is the correct place.

I have been through Plex and Jellyfin and have arrived at Navidrome for my music collection. I run
it in a Docker container and it's awesome. Also works perfectly with Symfonium for access in the car or mobile.

I would like to request a small feature. When you open a page to search through a music library, it would be extremely helpful to have a clickable alphabetical index bar or menu for album titles, artists, songs, etc. This would be a quick way to hit a page with a specific item vice doing a search or scrolling through pages. The attached photo has an example from an Albums page on the Jellyfin web site.

This index runs down the side of the Albums page view.

My reasoning: I have a collection of about 2000 digital albums. I frequently want to find a specific item and I nearly always know the title of the album. Clicking on the letter to bring the user to the page with those albums is often a lot faster and more efficient than typing a word or works into a search box.

Thanks for considering this. Love this app and its focus on just the music. 

![jellyfin-alpha](https://github.com/user-attachments/assets/1c98f638-1bfc-4497-a510-c86a33318e6a)

**Comment by NeuroDawg** [comment]
Came to request the same thing. This is a must have for those of use with large libraries.

**Comment by deluan** [comment]
I also have a sizeable library (6000 albums) but I personally prefer to use the search bar and type a few characters there. 

Anyway, this is something that we are considering for the new UI.

**Comment by joe-dougherty** [comment]
Deluan,

Thanks. Love all your work. Also works great on mobile with Symfonium.

Regards,

Joe
Joe Dougherty
Fleming Island, FL
Sent via mobile

**Comment by anayrat** [comment]
Hello,
I have the same request.

**Comment by Mundstrom** [comment]
I can't believe we had to request this. You'd think the devs themselves got tired of clicking "next" 30 times to get to albums starting with the letter "P" - it was the very first annoyance I encountered.
